src/intern_blks/ya_intern.c

```diff
@@ -129,6 +131,71 @@ __attribute__ ((gnu_inline)) inline void ya_setup_prefix_suffix(ya_block_t *blk,
 	}
 }
 
+#include <playerctl/playerctl.h>
+#define CHECK_PLAYER_ERROR(PREFIX) \
+	if (e_##PREFIX) { \
```

Comment:
I think just leaving the variable as `PREFIX` and calling the macro with the full variable name makes the code more legible.